homeassistant/components/twinkly/light.py

```diff
@@ -96,6 +100,7 @@ def __init__(
         self._attributes: dict[Any, Any] = {}
         self._current_movie: dict[Any, Any] = {}
         self._movies: list[Any] = []
+        self._attr_supported_features = LightEntityFeature.EFFECT
```

Comment:
You should set this by checking the firmware version against `MIN_EFFECT_VERSION` in `async_added_to_hass` life cycle.

Reply:
Fixed